@kiva/kv-components/vue/stories/KvCarousel.stories.js

```diff
@@ -199,3 +199,22 @@ export const loadingLoanCardExample = () => ({
 		},
 	},
 });
+
+export const ResponsiveSlides = () => ({
```

Comment:
This is the use case we are trying to solve. We want to display 2 slides at a time, at a fixed width that is approximately 50%.  But when we get to smaller resolutions, we want to display a single slide, that is responsive.